web/client/reducers/print.js

```diff
@@ -77,6 +79,10 @@ function print(state = {spec: initialSpec, capabilities: null, map: null, isLoad
         }
         return state;
     }
+    case ADD_PRINT_TRANSFORMER: {
+        addTransformer(action.name, action.transformer, action.position);
```

Comment:
Theoretically this is a side effect, and so it should go in an epic. reducers should be pure functions.

Reply:
I don't find any epics for print module in the [epcis folder](https://github.com/geosolutions-it/MapStore2/tree/master/web/client/epics).
Should I create a new epics file specially for this one ?
(I prefer asking you before pushing some modification).

thank you for your feedback !